### PR TITLE
Improved documentation for Processing Development Environment (PDE)

### PR DESCRIPTION
**Pull request by Arya-Gupta**
Issue: https://github.com/processing/processing-website/issues/422

Fixed grammatical errors in the [environment page](https://processing.org/environment).

Signed-off-by: Arya Gupta <arya.gupta99999@gmail.com>

### CODE REVIEW

File: content/pages/environment/index.mdx [hunk 1]
```diff
@@ -15,19 +15,19 @@ description: >
...
 
-Sketches can draw two- and three-dimensional graphics. The default renderer is for drawing two-dimensional graphics. The P3D renderer makes it possible to draw three-dimensional graphics, which includes controlling the camera, lighting, and materials. The P2D renderer is a fast, but less accurate renderer for drawing two-dimensional graphics. Both the P2D and P3D renderers are accelerated if your computer has an OpenGL compatible graphics card.
+Sketches can draw two- and three-dimensional graphics. The default renderer is for drawing two-dimensional graphics. The P3D renderer makes it possible to draw three-dimensional graphics, which include controlling the camera, lighting, and materials. The P2D renderer is a fast, but less accurate renderer for drawing two-dimensional graphics. Both the P2D and P3D renderers are accelerated if your computer has an OpenGL compatible graphics card.
```
Comment (SableRaf) [inline content/pages/environment/index.mdx:18]: I believe this "includes" refers to the ability of P3D to render 3D graphics and not to the phrase "three-dimensional graphics" itself, and therefore should keep its 's'

File: content/pages/environment/index.mdx [hunk 2]
```diff
@@ -341,7 +341,7 @@ Additional commands are found within the six menus: File, Edit, Sketch, Debug, T
 
 ## Preferences
 
-The Processing Development Environment (PDE) is highly configurable. The most common preferences can be modified in the Preferences window, located in the File menu on Windows and Linux and in the Processing menu on Mac Os X. The full list of preferences are stored in the "preferences.txt" file. This file can be opened and edited directly only when Processing is not running. You can find the location of this file on your computer by reading the bottom-left corner of the Preferences window.
+The Processing Development Environment (PDE) is highly configurable. The most common preferences can be modified in the Preferences window, located in the File menu on Windows and Linux and the Processing menu on Mac Os X. The full list of preferences is stored in the "preferences.txt" file. This file can be opened and edited directly only when Processing is not running. You can find the location of this file on your computer by reading the bottom-left corner of the Preferences window.
```
Comment (SableRaf) [inline content/pages/environment/index.mdx:344]: 'macOS' is not called X anymore. Just 'macOS'

**Review by SableRaf — CHANGES_REQUESTED** [review]
Added a couple of small fixes. Otherwise all good! Thanks for your contribution :)

**Comment by Arya-Gupta** [comment]
@SableRaf updated!

File: content/pages/environment/index.mdx [hunk 3]
```diff
@@ -99,7 +99,7 @@ Additional commands are found within the six menus: File, Edit, Sketch, Debug, T
   <dt>Preferences...</dt>
   <dd>
     Change some of the ways Processing works. (This item is located in the
-    Processing menu on Mac OS X.)
+    Processing menu on Mac OS.)
```
Comment (SableRaf) [inline content/pages/environment/index.mdx:102]: Thanks for the update! Small nitpick: it should be styled as **macOS**

**Review by SableRaf — CHANGES_REQUESTED** [review]

File: content/pages/environment/index.mdx (hunk 2 above)
Comment (SableRaf) [reply content/pages/environment/index.mdx:344]: @Arya-Gupta there is still this one thing pending. After that I'll happily merge your changes :)

**Review by SableRaf — APPROVED** [review]
Great, thanks for your contribution! ✨

**Pull request merged**